<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMain</class>
 <widget class="QDialog" name="DlgMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>手写板</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/image/handswriting.png</normaloff>:/image/handswriting.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>11</y>
     <width>304</width>
     <height>404</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Btn_clear">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>重写</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_delete">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_delete_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="Btn_word4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="Btn_word3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="Btn_word5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="Btn_word6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="Btn_word1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="Btn_word7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #4D4D4D,stop:1 #292929);
}
QPushButton{color:#4D4D4D;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="Btn_word2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="Btn_word9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="Btn_word8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_title" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>431</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>811</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(47, 47, 47);</string>
   </property>
  </widget>
  <zorder>widget_title</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
